content/en/data-playground/dc-metro.md

```diff
@@ -17,6 +17,8 @@ is_api_key_required: true
 api_type: JSON, GTFS protocol buffers
 sdks: N/A
 limits: The default tier (free) is rate-limited to 10 calls/second and 50,000 calls per day.
+diagram_image:
```

Comment:
Added to all of the content files to make it easier once the diagrams for each data source is available.